package.json

```diff
@@ -381,7 +381,7 @@
     "cross-spawn": "7.0.3",
     "csstype": "2.6.13",
     "csv-parse": "4.16.0",
-    "cypress": "10.2.0",
+    "cypress": "10.3.1",
```

Comment:
Make sure to also bump the Cypress version in the Dockerfile https://github.com/island-is/island.is/blob/main/scripts/ci/Dockerfile#L167